### DISCUSSION TITLE
Feature Request: Stat Background Color with Different Values

### DISCUSSION THREAD

**Discussion by Mueke666** [grafana/grafana#39426] [Data Sources]
Hello Everybody,

i have a Feature Request for the Stats Panel.
I want to Display a value and cloloring the background with a other value (1=Yellow or 0=Whit) from a different Table.

Thank You and greetings from Germany.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
